bears/general/LineCountBear.py

```diff
@@ -18,24 +18,37 @@ def _get_blank_line_count(self, file):
         return num_blank_lines
 
     def run(self, filename, file, max_lines_per_file: int,
-            exclude_blank_lines: bool=False):
+            exclude_blank_lines: bool=False, min_lines_per_file: int=0):
         """
-        Count the number of lines in a file and ensure that they are
-        smaller than a given size.
+        Count the number of lines in a file and ensure that they lie
+        within the range of given sizes.
```

Comment:
We need better doc string. This implies it is mandatory to provide the extremes while that is not the case